### PR TITLE
DRDT-85: Prebid

### PR DESCRIPTION
**Pull request by subinviswanathan**
[DRDT-85](https://readersdigest.atlassian.net/browse/DRDT-85)

### CODE REVIEW

**Comment by fariasf** [comment]
Created new staging environment for pr-66. Here are some useful links:

[WordPress Admin](https://pr-66-drdt.pantheonsite.io/wp/wp-admin)

[Homepage](https://pr-66-drdt.pantheonsite.io/)
[Article](https://pr-66-drdt.pantheonsite.io/jobsite-tips/abandoned-cable-removal/)
[Listicle](https://pr-66-drdt.pantheonsite.io/jobsite-tips/exterior/12-scaffolding-safety-tips-and-handy-hints/)
[Page](https://pr-66-drdt.pantheonsite.io/about-construction-pro-tips/)
[Hub](https://pr-66-drdt.pantheonsite.io/jobsite-tips/)

**Comment by fariasf** [comment]
There are visual regressions! 

View the [full visual regression test report](https://circleci.com/gh/fariasf/drdt-test/2195/artifacts/0/artifacts/backstop_data/html_report/index.html)

**Comment by fariasf** [comment]
⚠ Frontend performance tests failed!

Full Lighthouse performance results:

| Page/Branch | pr-66 | master | Result |
| --- | ---: | ---: | :---: |
| Homepage | [87](https://circleci.com/gh/fariasf/drdt-test/2196/artifacts/0/artifacts/lighthouse_results/pr-66/homepage/lighthouse.report.html) | [96](https://circleci.com/gh/fariasf/drdt-test/2196/artifacts/0/artifacts/lighthouse_results/master/homepage/lighthouse.report.html) | ✘ |
| Article | [97](https://circleci.com/gh/fariasf/drdt-test/2196/artifacts/0/artifacts/lighthouse_results/pr-66/article/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2196/artifacts/0/artifacts/lighthouse_results/master/article/lighthouse.report.html) | ✔ |
| Listicle | [85](https://circleci.com/gh/fariasf/drdt-test/2196/artifacts/0/artifacts/lighthouse_results/pr-66/listicle/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2196/artifacts/0/artifacts/lighthouse_results/master/listicle/lighthouse.report.html) | ✘ |
| Page | [98](https://circleci.com/gh/fariasf/drdt-test/2196/artifacts/0/artifacts/lighthouse_results/pr-66/page/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2196/artifacts/0/artifacts/lighthouse_results/master/page/lighthouse.report.html) | ✔ |
| Hub | [95](https://circleci.com/gh/fariasf/drdt-test/2196/artifacts/0/artifacts/lighthouse_results/pr-66/hub/lighthouse.report.html) | [96](https://circleci.com/gh/fariasf/drdt-test/2196/artifacts/0/artifacts/lighthouse_results/master/hub/lighthouse.report.html) | ✔ |


(**Note: Tests are against the _NOADS_ version of the pages!**)

**Comment by fariasf** [comment]
Updated staging environment for pr-66. Here are some useful links:

[WordPress Admin](https://pr-66-drdt.pantheonsite.io/wp/wp-admin)

[Homepage](https://pr-66-drdt.pantheonsite.io/)
[Article](https://pr-66-drdt.pantheonsite.io/jobsite-tips/abandoned-cable-removal/)
[Listicle](https://pr-66-drdt.pantheonsite.io/jobsite-tips/exterior/12-scaffolding-safety-tips-and-handy-hints/)
[Page](https://pr-66-drdt.pantheonsite.io/about-construction-pro-tips/)
[Hub](https://pr-66-drdt.pantheonsite.io/jobsite-tips/)

**Comment by fariasf** [comment]
There are visual regressions! 

View the [full visual regression test report](https://circleci.com/gh/fariasf/drdt-test/2247/artifacts/0/artifacts/backstop_data/html_report/index.html)

**Comment by fariasf** [comment]
⚠ Frontend performance tests failed!

Full Lighthouse performance results:

| Page/Branch | pr-66 | master | Result |
| --- | ---: | ---: | :---: |
| Homepage | [63](https://circleci.com/gh/fariasf/drdt-test/2246/artifacts/0/artifacts/lighthouse_results/pr-66/homepage/lighthouse.report.html) | [97](https://circleci.com/gh/fariasf/drdt-test/2246/artifacts/0/artifacts/lighthouse_results/master/homepage/lighthouse.report.html) | ✘ |
| Article | [97](https://circleci.com/gh/fariasf/drdt-test/2246/artifacts/0/artifacts/lighthouse_results/pr-66/article/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2246/artifacts/0/artifacts/lighthouse_results/master/article/lighthouse.report.html) | ✔ |
| Listicle | [98](https://circleci.com/gh/fariasf/drdt-test/2246/artifacts/0/artifacts/lighthouse_results/pr-66/listicle/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2246/artifacts/0/artifacts/lighthouse_results/master/listicle/lighthouse.report.html) | ✔ |
| Page | [98](https://circleci.com/gh/fariasf/drdt-test/2246/artifacts/0/artifacts/lighthouse_results/pr-66/page/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2246/artifacts/0/artifacts/lighthouse_results/master/page/lighthouse.report.html) | ✔ |
| Hub | [97](https://circleci.com/gh/fariasf/drdt-test/2246/artifacts/0/artifacts/lighthouse_results/pr-66/hub/lighthouse.report.html) | [97](https://circleci.com/gh/fariasf/drdt-test/2246/artifacts/0/artifacts/lighthouse_results/master/hub/lighthouse.report.html) | ✔ |


(**Note: Tests are against the _NOADS_ version of the pages!**)

**Comment by fariasf** [comment]
Updated staging environment for pr-66. Here are some useful links:

[WordPress Admin](https://pr-66-drdt.pantheonsite.io/wp/wp-admin)

[Homepage](https://pr-66-drdt.pantheonsite.io/)
[Article](https://pr-66-drdt.pantheonsite.io/jobsite-tips/abandoned-cable-removal/)
[Listicle](https://pr-66-drdt.pantheonsite.io/jobsite-tips/exterior/12-scaffolding-safety-tips-and-handy-hints/)
[Page](https://pr-66-drdt.pantheonsite.io/about-construction-pro-tips/)
[Hub](https://pr-66-drdt.pantheonsite.io/jobsite-tips/)

File: web/wp-content/plugins/tmbi-prebid/js/tmbi-prebid.js [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+	if ( typeof prebid_conf !== 'undefined' && prebid_conf['openx_blocked_flag'] ) {
+		console.warn('OpenX blocked from url');
+		return prebid_ad;
+	}
+	if ( ! ( options.openx_del_domain ) ) {
+		console.warn( 'Ad Stack', 'Prebid', 'OpenX is not configured / is not enabled.' );
+		return prebid_ad;
+	}
+	console.log( 'Ad Stack', 'Prebid', 'OpenX call for', ad_slot );
+	/* @todo: Find a sane way to make this matrix configurable */
```
Comment (fariasf) [inline web/wp-content/plugins/tmbi-prebid/js/tmbi-prebid.js:111]: This PR seems a good time to remove the TOH, RD and FHM code, and to create a ticket to fix this. Is there one?

**Comment by fariasf** [comment]
There are visual regressions! 

View the [full visual regression test report](https://circleci.com/gh/fariasf/drdt-test/2313/artifacts/0/artifacts/backstop_data/html_report/index.html)

**Comment by fariasf** [comment]
⚠ Frontend performance tests failed!

Full Lighthouse performance results:

| Page/Branch | pr-66 | master | Result |
| --- | ---: | ---: | :---: |
| Homepage | [87](https://circleci.com/gh/fariasf/drdt-test/2312/artifacts/0/artifacts/lighthouse_results/pr-66/homepage/lighthouse.report.html) | [96](https://circleci.com/gh/fariasf/drdt-test/2312/artifacts/0/artifacts/lighthouse_results/master/homepage/lighthouse.report.html) | ✘ |
| Article | [87](https://circleci.com/gh/fariasf/drdt-test/2312/artifacts/0/artifacts/lighthouse_results/pr-66/article/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2312/artifacts/0/artifacts/lighthouse_results/master/article/lighthouse.report.html) | ✘ |
| Listicle | [98](https://circleci.com/gh/fariasf/drdt-test/2312/artifacts/0/artifacts/lighthouse_results/pr-66/listicle/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2312/artifacts/0/artifacts/lighthouse_results/master/listicle/lighthouse.report.html) | ✔ |
| Page | [98](https://circleci.com/gh/fariasf/drdt-test/2312/artifacts/0/artifacts/lighthouse_results/pr-66/page/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2312/artifacts/0/artifacts/lighthouse_results/master/page/lighthouse.report.html) | ✔ |
| Hub | [97](https://circleci.com/gh/fariasf/drdt-test/2312/artifacts/0/artifacts/lighthouse_results/pr-66/hub/lighthouse.report.html) | [95](https://circleci.com/gh/fariasf/drdt-test/2312/artifacts/0/artifacts/lighthouse_results/master/hub/lighthouse.report.html) | ✔ |


(**Note: Tests are against the _NOADS_ version of the pages!**)

**Comment by fariasf** [comment]
Updated staging environment for pr-66. Here are some useful links:

[WordPress Admin](https://pr-66-drdt.pantheonsite.io/wp/wp-admin)

[Homepage](https://pr-66-drdt.pantheonsite.io/)
[Article](https://pr-66-drdt.pantheonsite.io/jobsite-tips/abandoned-cable-removal/)
[Listicle](https://pr-66-drdt.pantheonsite.io/jobsite-tips/exterior/12-scaffolding-safety-tips-and-handy-hints/)
[Page](https://pr-66-drdt.pantheonsite.io/about-construction-pro-tips/)
[Hub](https://pr-66-drdt.pantheonsite.io/jobsite-tips/)

**Comment by fariasf** [comment]
There are visual regressions! 

View the [full visual regression test report](https://circleci.com/gh/fariasf/drdt-test/2320/artifacts/0/artifacts/backstop_data/html_report/index.html)

**Comment by fariasf** [comment]
Congrats! Frontend performance tests passed!

Full Lighthouse performance results:

| Page/Branch | pr-66 | master | Result |
| --- | ---: | ---: | :---: |
| Homepage | [93](https://circleci.com/gh/fariasf/drdt-test/2319/artifacts/0/artifacts/lighthouse_results/pr-66/homepage/lighthouse.report.html) | [96](https://circleci.com/gh/fariasf/drdt-test/2319/artifacts/0/artifacts/lighthouse_results/master/homepage/lighthouse.report.html) | ✔ |
| Article | [98](https://circleci.com/gh/fariasf/drdt-test/2319/artifacts/0/artifacts/lighthouse_results/pr-66/article/lighthouse.report.html) | [97](https://circleci.com/gh/fariasf/drdt-test/2319/artifacts/0/artifacts/lighthouse_results/master/article/lighthouse.report.html) | ✔ |
| Listicle | [97](https://circleci.com/gh/fariasf/drdt-test/2319/artifacts/0/artifacts/lighthouse_results/pr-66/listicle/lighthouse.report.html) | [97](https://circleci.com/gh/fariasf/drdt-test/2319/artifacts/0/artifacts/lighthouse_results/master/listicle/lighthouse.report.html) | ✔ |
| Page | [98](https://circleci.com/gh/fariasf/drdt-test/2319/artifacts/0/artifacts/lighthouse_results/pr-66/page/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2319/artifacts/0/artifacts/lighthouse_results/master/page/lighthouse.report.html) | ✔ |
| Hub | [97](https://circleci.com/gh/fariasf/drdt-test/2319/artifacts/0/artifacts/lighthouse_results/pr-66/hub/lighthouse.report.html) | [94](https://circleci.com/gh/fariasf/drdt-test/2319/artifacts/0/artifacts/lighthouse_results/master/hub/lighthouse.report.html) | ✔ |


(**Note: Tests are against the _NOADS_ version of the pages!**)

File: web/wp-content/plugins/tmbi-prebid/js/tmbi-prebid.js (hunk 1 above)
Comment (subinviswanathan) [reply web/wp-content/plugins/tmbi-prebid/js/tmbi-prebid.js:111]: @fariasf  Yes i think of moving this to settings page for prebid.
Till now didnt create. will have to create one.

Comment (subinviswanathan) [reply web/wp-content/plugins/tmbi-prebid/js/tmbi-prebid.js:111]: @fariasf https://readersdigest.atlassian.net/browse/DRDT-104 created

**Comment by fariasf** [comment]
Updated staging environment for pr-66. Here are some useful links:

[WordPress Admin](https://pr-66-drdt.pantheonsite.io/wp/wp-admin)

[Homepage](https://pr-66-drdt.pantheonsite.io/)
[Article](https://pr-66-drdt.pantheonsite.io/jobsite-tips/abandoned-cable-removal/)
[Listicle](https://pr-66-drdt.pantheonsite.io/jobsite-tips/exterior/12-scaffolding-safety-tips-and-handy-hints/)
[Page](https://pr-66-drdt.pantheonsite.io/about-construction-pro-tips/)
[Hub](https://pr-66-drdt.pantheonsite.io/jobsite-tips/)

**Comment by fariasf** [comment]
There are visual regressions! 

View the [full visual regression test report](https://circleci.com/gh/fariasf/drdt-test/2504/artifacts/0/artifacts/backstop_data/html_report/index.html)

**Comment by fariasf** [comment]
⚠ Frontend performance tests failed!

Full Lighthouse performance results:

| Page/Branch | pr-66 | master | Result |
| --- | ---: | ---: | :---: |
| Homepage | [94](https://circleci.com/gh/fariasf/drdt-test/2505/artifacts/0/artifacts/lighthouse_results/pr-66/homepage/lighthouse.report.html) | [96](https://circleci.com/gh/fariasf/drdt-test/2505/artifacts/0/artifacts/lighthouse_results/master/homepage/lighthouse.report.html) | ✔ |
| Article | [98](https://circleci.com/gh/fariasf/drdt-test/2505/artifacts/0/artifacts/lighthouse_results/pr-66/article/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2505/artifacts/0/artifacts/lighthouse_results/master/article/lighthouse.report.html) | ✔ |
| Listicle | [98](https://circleci.com/gh/fariasf/drdt-test/2505/artifacts/0/artifacts/lighthouse_results/pr-66/listicle/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2505/artifacts/0/artifacts/lighthouse_results/master/listicle/lighthouse.report.html) | ✔ |
| Page | [98](https://circleci.com/gh/fariasf/drdt-test/2505/artifacts/0/artifacts/lighthouse_results/pr-66/page/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2505/artifacts/0/artifacts/lighthouse_results/master/page/lighthouse.report.html) | ✔ |
| Hub | [88](https://circleci.com/gh/fariasf/drdt-test/2505/artifacts/0/artifacts/lighthouse_results/pr-66/hub/lighthouse.report.html) | [95](https://circleci.com/gh/fariasf/drdt-test/2505/artifacts/0/artifacts/lighthouse_results/master/hub/lighthouse.report.html) | ✘ |


(**Note: Tests are against the _NOADS_ version of the pages!**)

**Comment by fariasf** [comment]
Updated staging environment for pr-66. Here are some useful links:

[WordPress Admin](https://pr-66-drdt.pantheonsite.io/wp/wp-admin)

[Homepage](https://pr-66-drdt.pantheonsite.io/)
[Article](https://pr-66-drdt.pantheonsite.io/jobsite-tips/abandoned-cable-removal/)
[Listicle](https://pr-66-drdt.pantheonsite.io/jobsite-tips/exterior/12-scaffolding-safety-tips-and-handy-hints/)
[Page](https://pr-66-drdt.pantheonsite.io/about-construction-pro-tips/)
[Hub](https://pr-66-drdt.pantheonsite.io/jobsite-tips/)

**Comment by fariasf** [comment]
There are visual regressions! 

View the [full visual regression test report](https://circleci.com/gh/fariasf/drdt-test/2531/artifacts/0/artifacts/backstop_data/html_report/index.html)

**Comment by fariasf** [comment]
Congrats! Frontend performance tests passed!

Full Lighthouse performance results:

| Page/Branch | pr-66 | master | Result |
| --- | ---: | ---: | :---: |
| Homepage | [94](https://circleci.com/gh/fariasf/drdt-test/2532/artifacts/0/artifacts/lighthouse_results/pr-66/homepage/lighthouse.report.html) | [96](https://circleci.com/gh/fariasf/drdt-test/2532/artifacts/0/artifacts/lighthouse_results/master/homepage/lighthouse.report.html) | ✔ |
| Article | [98](https://circleci.com/gh/fariasf/drdt-test/2532/artifacts/0/artifacts/lighthouse_results/pr-66/article/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2532/artifacts/0/artifacts/lighthouse_results/master/article/lighthouse.report.html) | ✔ |
| Listicle | [98](https://circleci.com/gh/fariasf/drdt-test/2532/artifacts/0/artifacts/lighthouse_results/pr-66/listicle/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2532/artifacts/0/artifacts/lighthouse_results/master/listicle/lighthouse.report.html) | ✔ |
| Page | [98](https://circleci.com/gh/fariasf/drdt-test/2532/artifacts/0/artifacts/lighthouse_results/pr-66/page/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2532/artifacts/0/artifacts/lighthouse_results/master/page/lighthouse.report.html) | ✔ |
| Hub | [97](https://circleci.com/gh/fariasf/drdt-test/2532/artifacts/0/artifacts/lighthouse_results/pr-66/hub/lighthouse.report.html) | [94](https://circleci.com/gh/fariasf/drdt-test/2532/artifacts/0/artifacts/lighthouse_results/master/hub/lighthouse.report.html) | ✔ |


(**Note: Tests are against the _NOADS_ version of the pages!**)

**Comment by fariasf** [comment]
Updated staging environment for pr-66. Here are some useful links:

[WordPress Admin](https://pr-66-drdt.pantheonsite.io/wp/wp-admin)

[Homepage](https://pr-66-drdt.pantheonsite.io/)
[Article](https://pr-66-drdt.pantheonsite.io/jobsite-tips/abandoned-cable-removal/)
[Listicle](https://pr-66-drdt.pantheonsite.io/jobsite-tips/exterior/12-scaffolding-safety-tips-and-handy-hints/)
[Page](https://pr-66-drdt.pantheonsite.io/about-construction-pro-tips/)
[Hub](https://pr-66-drdt.pantheonsite.io/jobsite-tips/)

**Comment by fariasf** [comment]
There are visual regressions! 

View the [full visual regression test report](https://circleci.com/gh/fariasf/drdt-test/2545/artifacts/0/artifacts/backstop_data/html_report/index.html)

**Comment by fariasf** [comment]
Congrats! Frontend performance tests passed!

Full Lighthouse performance results:

| Page/Branch | pr-66 | master | Result |
| --- | ---: | ---: | :---: |
| Homepage | [94](https://circleci.com/gh/fariasf/drdt-test/2546/artifacts/0/artifacts/lighthouse_results/pr-66/homepage/lighthouse.report.html) | [96](https://circleci.com/gh/fariasf/drdt-test/2546/artifacts/0/artifacts/lighthouse_results/master/homepage/lighthouse.report.html) | ✔ |
| Article | [98](https://circleci.com/gh/fariasf/drdt-test/2546/artifacts/0/artifacts/lighthouse_results/pr-66/article/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2546/artifacts/0/artifacts/lighthouse_results/master/article/lighthouse.report.html) | ✔ |
| Listicle | [98](https://circleci.com/gh/fariasf/drdt-test/2546/artifacts/0/artifacts/lighthouse_results/pr-66/listicle/lighthouse.report.html) | [97](https://circleci.com/gh/fariasf/drdt-test/2546/artifacts/0/artifacts/lighthouse_results/master/listicle/lighthouse.report.html) | ✔ |
| Page | [98](https://circleci.com/gh/fariasf/drdt-test/2546/artifacts/0/artifacts/lighthouse_results/pr-66/page/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2546/artifacts/0/artifacts/lighthouse_results/master/page/lighthouse.report.html) | ✔ |
| Hub | [97](https://circleci.com/gh/fariasf/drdt-test/2546/artifacts/0/artifacts/lighthouse_results/pr-66/hub/lighthouse.report.html) | [93](https://circleci.com/gh/fariasf/drdt-test/2546/artifacts/0/artifacts/lighthouse_results/master/hub/lighthouse.report.html) | ✔ |


(**Note: Tests are against the _NOADS_ version of the pages!**)

**Pull request merged**